.circleci/config.yml

```diff
@@ -378,7 +378,8 @@ commands:
           command: |
             echo 'export API_BASE_URL=$API_BASE_URL_AWS_CONTENT' >> $BASH_ENV
             echo 'export AUTH_DOMAIN=$COGNITO_AUTH_DOMAIN_CONTENT' >> $BASH_ENV
-
+            echo 'export MYNJ_SERVICE_TOKEN=$MYNJ_SERVICE_TOKEN_CONTENT' >> $BASH_ENV
+            echo 'export MYNJ_SERVICE_TOKEN=$MYNJ_SERVICE_TOKEN_CONTENT' >> $BASH_ENV
```

Comment:
[boulder] This appears duplicated. Did you mean to instead add the MYNJ_ROLE_NAME and MYNJ_SERVICE_URL fields?

Reply:
I meant to add the MYNJ_ROLE_NAME. With the MYNJ_SERVICE_URL if they are already using what's in dev. So we do not need to set it again. what do you think >